### DISCUSSION TITLE
Listbox menu slow

### DISCUSSION THREAD

**Discussion by Dragomir-Ivanov** [heroui-inc/heroui#2542] [Feedback]
Greetings,
I am evaluating NextUI for my next project, but I saw one red flag. The Select component menu item highlight seems slow to track the cursor movement. Granted my laptop is not top-notch, but this issue is not on the Dropdown menu. Dropdown menu seems to be snappy. I have `disableAnimation` for each `SelectItem`, and disabled `scrollShadow` for Select menu, still laggy.
I see this on the NextUI documentation site, as well as on my Vite bootstrapped project.

Does anybody feel the same, and are there any workarounds?

**Comment by TudorEsan** [comment]
same here

**Comment by usersina** [comment]
I've tried both [Dropdown](https://nextui.org/docs/components/dropdown) and [Select](https://nextui.org/docs/components/select) with just 114 items, and it's already lagging. I think it's because of [this issue](https://github.com/radix-ui/primitives/issues/2356).
